app/controllers/saml_controller.rb

```diff
@@ -4,6 +4,7 @@ class SamlController < ApplicationController
 
   def metadata
     meta = OneLogin::RubySaml::Metadata.new
-    render xml: meta.generate(SAML::SETTINGS), content_type: 'application/xml'
+    saml_settings
+    render xml: meta.generate(saml_settings), content_type: 'application/xml'
```

Comment:
why does saml_settings need to get called twice, can we get rid of the first one and just use it in the meta.generate the way you have in the second line?